posts/inside-rust/2021-06-15-boxyuwu-leseulartichaut-the8472-compiler-contributors.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Please welcome Boxy, Léo Lanteri Thauvin and the8472 to compiler-contributors"
+author: Wesley Wiser
+description: "Please welcome Boxy, Léo Lanteri Thauvin and the8472 to compiler-contributors"
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+Please welcome [Boxy], [Léo Lanteri Thauvin] and [the8472] to the [compiler-contributors] group!
+
+[Boxy] has been working on pushing const generics forward by [implementing parts] of the `const_evaluatable_checked` feature, [fixing bugs] and [making rustdoc work with const generics].
+Boxy is also a frequent contributor to [#project-const-generics] discussions and meetings.
+
+[Léo Lanteri Thauvin] has been contributing refactorings and improvements to rustc for several years now.
+Recently, Léo has been [implementing] a [Major Change Proposal] to migrate the `unsafe` checker to operate on HIR instead of MIR.
```

Comment:
```suggestion
Recently, Léo has been [implementing] a [Major Change Proposal] to migrate the `unsafe` checker to operate on THIR instead of MIR.
```